analyzers/tests/SonarAnalyzer.Test/TestCases/LoggingTemplatePlaceHoldersShouldBeInOrder.cs

```diff
@@ -103,7 +113,15 @@ public void PropertyAccess(ILogger logger, Person person, List<Person> people)
         logger.LogInformation("People: {Count} {People}", people.Count, people.Select(x => x.FirstName + " " + x.LastName));
 
         var someFirstName = person.FirstName;
-        logger.LogInformation("{PersonFirstName} {PersonAge}", someFirstName, person.Age);  // Compliant
+        logger.LogInformation("{PersonFirstName} {PersonAge}", someFirstName, person.Age);      // Compliant
+
+        var first = person.FirstName;
+        var last = person.LastName;
+        var name = person.LastName;
+        logger.LogInformation("Person: {FirstName} {LastName}", first, last);       // Compliant
+        logger.LogInformation("Person: {FirstName} {LastName}", last, first);       // FN
+        logger.LogInformation("Person: {FirstName} {LastName}", last, name);        // FN
+        logger.LogInformation("Person: {FirstName} {LastName}", name, name);        // FN
```

Comment:
I think these are not really FNs. Just cases where the intention is not clear and we cannot tell.